File: 1-two-sum/1-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        for i in nums:
            temp = list(nums)
            temp.remove(i)
            for j in temp:
                if i+j == target:
                    return [nums.index(i),nums.index(j,nums.index(i)+1)]

File: 1-two-sum/1-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int m,n;
        vector<int> v;
        for(int i=0;i<nums.size();i++)
        {
            for(int j=i;j<nums.size();j++)
            {
               
                 if(nums[i]+nums[j]==target and i!=j)
                {
                    v.push_back(i);
                    v.push_back(j);
                }
            }
        }
        return v;
        
    }
};